src/components/SelectComponents/Autosuggest/SuggestionTag/__tests__/SuggestionTag.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { SuggestionTag } from '../SuggestionTag';
+
+describe('SuggestionTag', () => {
+    it('should render nothing if no children are provided', () => {
+        const view = render(<SuggestionTag />);
+
+        expect(view.container).toMatchSnapshot();
```

Comment:
there is no expectation to verify if nothing was rendered.

Reply:
:+1: